### ISSUE TITLE
Include pprint() in utility functions

### ISSUE THREAD

**Issue by rlafuente** [shoebot/shoebot#419]
It's super handy for reading through lists and dicts, and i think it makes sense to have it close at hand.

**Comment by stuaxo** [comment]
Do you mean, have this inside the shoebot namespace for bots ?

I guess ?   Though it most people that know of it will be used to `from pprint import pprint`.

**Comment by rlafuente** [comment]
Yeah exactly! And yes, it's an import away, but I was looking into how Processing wraps functions like sin() and cos() this way, and at least to me that is useful :)

**Comment by stuaxo** [comment]
As long as the docs mention the equivalent in plain python I think it's good.

OT:  We should probably think twice about changing the API of things that are used commonly in python (e.g. nodeboxes version of random() vs math.random() have different calling conventions.
